staging/src/k8s.io/cloud-provider/controllers/service/controller.go

```diff
@@ -552,19 +552,15 @@ func needsCleanup(service *v1.Service) bool {
 }
 
 // needsUpdate checks if load balancer needs to be updated due to change in attributes.
-func (c *Controller) needsUpdate(oldService *v1.Service, newService *v1.Service) bool {
+func needsUpdate(oldService *v1.Service, newService *v1.Service) bool {
 	if !wantsLoadBalancer(oldService) && !wantsLoadBalancer(newService) {
 		return false
 	}
 	if wantsLoadBalancer(oldService) != wantsLoadBalancer(newService) {
-		c.eventRecorder.Eventf(newService, v1.EventTypeNormal, "Type", "%v -> %v",
```

Comment:
/hold
@carlory we need to keep the information, can you replace the events by logs? V(2) should be enough

Reply:
updated. @aojea 

Can you take a look at the release note for this PR? It is a bug fix for the cloud provider's cloud-controller-manager if they use the library `k8s.io/cloud-provider` version 0.31.0 or later.

Should we add an `Action Required` section to the release note?

Reply:
I think is worth to mention it but I don't know if is as urgent as an Action required thing